docs/howto/hooks/lua.md

```diff
@@ -231,6 +231,10 @@ Returns 2 values:
 1. The HTTP status code returned by the lakeFS API
 1. The content of the specified object as a lua string
 
+### `lakefs/diff_branch(repository_id, branch_id)`
```

Comment:
```suggestion
### `lakefs/diff_branch(repository_id, branch_id[, after, prefix, delimiter, amount])`
```

Reply:
@ozkatz , thank you for the input! I've now updated the documentation for the changes to include all the optional parameters.